src/field_fadetransition.c

```diff
@@ -610,7 +611,7 @@ void DoTeleportWarp(void)
     gFieldCallback = FieldCB_TeleportWarpIn;
 }
 
-void sub_807E5C4(void)
+static void DoTeleport2WarpWithoutCallback(void) // Unused
```

Comment:
This is `DoPortholeWarp` from pokeemerald (a lot of the functions here have names in pokeemerald, but most of these are better).